python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -5,6 +5,13 @@
 import numpy as np
 from scipy import interpolate as ipol
 
+import astropy
+from astropy import units as u
+from astropy.time import Time
+from astropy.coordinates import SkyCoord, EarthLocation, AltAz
+import astroplan
```

Comment:
An ``astroplan`` sub-module are imported explicitly later on. Is it necessary to import the top-level astroplan package?